src/Microsoft.AspNet.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -15,16 +15,16 @@
 namespace Microsoft.AspNet.JsonPatch.Adapters
 {
     /// <inheritdoc />
-    public class ObjectAdapter<TModel> : IObjectAdapter<TModel> where TModel : class
+    internal class ObjectAdapter : IObjectAdapter
```

Comment:
We don't use the `internal` keyword in our repos unless there's a very good reason to keep someone from using something. The general philosophy is to make it possible to use if a user wants to mock it for tests or new it up.

If you want to put it in a more obscure namespace like `Microsoft.AspNet.JsonPatch.Internal` that would be ok too, but I think it's fine as-is.